pkgs/development/libraries/science/math/tensorrt/generic.nix

```diff
@@ -0,0 +1,66 @@
+{ lib
+, stdenv
+, callPackage
+, cudatoolkit
+, cudaPackages
```

Comment:
```diff
- , cudaPackages
```

We have a convention that expressions inside `cudaPackages` scope ask directly for individual `cudaPackages` elements (like `cudatoolki`, `cuda_nvcc`, etc). In contrast, external expressions only ask for the entirety of `cudaPackages`